package.json

```
@@ -43,10 +43,10 @@
     "format-php": "composer format",
     "phpstan": "composer phpstan",
     "lint-php": "composer lint:all",
-    "test-php": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test",
-    "test-php-multisite": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test-multisite",
-    "test-php-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test:plugins",
-    "test-php-multisite-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/$(basename $(pwd)) composer test-multisite:plugins",
+    "test-php": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test",
+    "test-php-multisite": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test-multisite",
+    "test-php-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test:plugins",
+    "test-php-multisite-plugins": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test-multisite:plugins",
```

Is `test-php` now obsolete since `performance-lab` is tested as part of `test-php-plugins?

```suggestion
    "test-php": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test:plugins",
    "test-php-multisite": "wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test-multisite:plugins",
```

---

Might be required in case someone want to play around with different PHPUnit cli options?

---

Why? They can still pass them to to `test-php-plugins` just the same, right?

---

A developer would be confused why there are two sets of `test` commands. At least, I was confused.

---

If I just want to run a test in isolation for a single plugin only, it might be required IMO.

---

> A developer would be confused why there are two sets of test commands. At least, I was confused.

That is also true.

---

I think for that we should do something like what you've done in #1065. There needs to be an easy way to run tests for a single plugin via `npm`.

---

I think we can go with your suggestion for now. In the future, it can be iterated given we have multiple pending discussions regarding such behaviors.

---

I've just added a script locally to facilitate testing one or more plugins in isolation:

```bash
#!/bin/bash
for plugin in $@; do
	wp-env run tests-cli --env-cwd=/var/www/html/wp-content/plugins/performance composer test:$plugin
done
```

Usage:

```bash
bash bin/test-single-plugin.sh performance-lab auto-sizes
```

---

I like the consolidation here because I agree that running `npm run test-php` should run the full suite, but since we're now running a set of individual suites, you can't pass other flags to `npm run test-php`. I think we can iterate on this in another followup issue though since as @thelovekesh [mentioned](https://github.com/WordPress/performance/pull/1182#discussion_r1610502152) we haven't aligned on our PHPUnit behavior in general.